client/view/Form/Form.css

```diff
@@ -18,6 +18,7 @@
   background-color: var(--main-bg);
   border: none;
   border-radius: 8px;
+  resize: none;
```

Comment:
We can’t disable resize at all. Let’s allow only vertical resize and add min height.